dask/dataframe/io/sql.py

```diff
@@ -133,6 +158,14 @@ def read_sql_table(table, uri, index_col, divisions=None, npartitions=None,
         cond = index <= upper if i == len(lowers) - 1 else index < upper
         q = sql.select(columns).where(sql.and_(index >= lower, cond)
                                       ).select_from(table)
-        parts.append(delayed(pd.read_sql)(q, uri, **kwargs))
+        parts.append(delayed(_rationalize_sql)(q, uri, head[:0], **kwargs))
 
     return from_delayed(parts, head, divisions=divisions)
+
+
+def _rationalize_sql(q, uri, meta, **kwargs):
```

Comment:
This function name will show up in the graphs. Perhaps something more user friendly like `_read_sql_chunk`.